### PR TITLE
Display investigator image for deck lists

### PR DESCRIPTION
**Pull request by slyjeff**
This will make it much easier to immediatly identify which deck I am working with while streaming

### CODE REVIEW

File: ArkhamOverlay/Utils/IHasImageButton.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    public interface IHasImageButton {
+        string Name { get; }
+        CardType ImageCardType { get; }
+        ImageSource Image { get; set; }
+        ImageSource ButtonImage { get; set; }
+        byte[] ButtonImageAsBytes { get; set; }
+    }
+
+    public static class HasImageButtonExtensions {
+        private static readonly Dictionary<string, BitmapImage> CardImageCache = new Dictionary<string, BitmapImage>();
```
Comment (davilimap) [inline ArkhamOverlay/Utils/IHasImageButton.cs:18]: Minor: This isn't great IMO. If we're caching things, (especially big stuff like images) I'd rather have a dedicated class or server. This is probably fine for now, but would be good to make a note of it.

File: ArkhamOverlay/Utils/ImageUtils.cs [hunk 2]
```diff
@@ -23,5 +26,53 @@ public static class ImageUtils {
...
+                case CardType.Event:
+                    return new Point(40, 0);
+                case CardType.Skill:
+                    return new Point(40, 40);
+                default:
+                    return new Point(40, 40);
+            }
+        }
+
+        public static byte[] AsBytes(this ImageSource image) {
```
Comment (davilimap) [inline ArkhamOverlay/Utils/ImageUtils.cs:63]: Minor: Just returning null here isn't great. Maybe we should consider adding an "Image not found" graphic.

**Review by davilimap — APPROVED** [review]
Please read suggestions.

Also need to consider the logging PR. Either merge that one in or remember to add logging to this afterwards.

File: ArkhamOverlay/Utils/ImageUtils.cs (hunk 2 above)
Comment (slyjeff) [reply ArkhamOverlay/Utils/ImageUtils.cs:63]: I added a todo comment for this

File: ArkhamOverlay/Utils/IHasImageButton.cs (hunk 1 above)
Comment (slyjeff) [reply ArkhamOverlay/Utils/IHasImageButton.cs:18]: Discussed- we'll consider implementing a better caching strategy as a feature in the future. I added an issue for this.

**Pull request merged**